### PR TITLE
Limit number of Python package versions

### PR DESCRIPTION
**Pull request by maximiliankolb**
Refs PR 11202 for katello on GitHub
Refs Remine issue 37974

#### What changes are you introducing?

image

#### Why are you introducing these changes? (Explanation, links to references, issues, etc.)

new option in katello nightly

#### Checklists

* [x] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by knoppi** [comment]
LGTM!

**Review by sbernhard — APPROVED** [review]

File: guides/common/modules/proc_synchronizing-python-repositories.adoc [hunk 1]
```diff
@@ -21,6 +21,8 @@ Clear this field if the repository does not require authentication.
 Clear this field if the repository does not require authentication.
 . Optional: In the *Excludes* field, enter a list of Python packages separated by newlines to exclude from synchronizing to {Project}.
 . Optional: In the *Includes* field, enter a list of Python packages separated by newlines to limit the synchronization only to the included packages.
+. Optional: In the *Keep latest packages* field, enter a number to limit the amount of Python packages kept after synchronization.
+By default, {Project} keeps all package versions.
```
Comment (Lennonka) [inline guides/common/modules/proc_synchronizing-python-repositories.adoc:25]: I'm wondering what the number means. How does it select the packages to keep? If I enter 2, will it keep two latest versions of each Python package?
```suggestion
. Optional: In the *Keep latest packages* field, enter a number to limit the amount of Python package versions kept after synchronization.
By default, {Project} keeps all package versions.
```

Comment (maximiliankolb) [reply guides/common/modules/proc_synchronizing-python-repositories.adoc]: Yes, it will keep the latest N versions per package. Applied your suggestion.

**Review by maximiliankolb — COMMENTED** [review]
Rebased to "master" and ready for re-review.

**Pull request merged**